File: airbyte-cdk/java/airbyte-cdk/dependencies/src/main/java/io/airbyte/commons/functional/CheckedBiConsumer.java
/*
 * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.commons.functional;

import org.apache.commons.lang3.function.FailableBiConsumer;

@FunctionalInterface
public interface CheckedBiConsumer<T, R, E extends Throwable> extends FailableBiConsumer<T, R, E> {

}

File: airbyte-cdk/java/airbyte-cdk/dependencies/bin/main/io/airbyte/commons/functional/CheckedBiConsumer.kt
/*
 * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 */
package io.airbyte.commons.functional

import org.apache.commons.lang3.function.FailableBiConsumer

fun interface CheckedBiConsumer<T, R, E : Throwable?> : FailableBiConsumer<T, R, E>
